site/components/src/main/java/uk/nhs/hee/web/utils/ReportAndPublicationUtils.java

```diff
@@ -42,4 +55,68 @@ public PublicationLandingPage findMyParent(final HippoDocumentBean reportBean, f
         }
         return null;
     }
+
+    /**
+     * Load values from the PublicationLandingPage and set up the maps which hold their keys/values
+     *
+     * Once found, the maps are added to the request object and can be used in the template
+     * @param request
+     * @param locale
+     * @param publicationLandingPage
+     */
+    public void addPublicationLandingPageTaxonomyFieldsToModel(final HstRequest request, final Locale locale, PublicationLandingPage publicationLandingPage) {
+        final TaxonomyManager taxonomyManager = HstServices.getComponentManager()
+                .getComponent(TaxonomyManager.class.getSimpleName(),"org.onehippo.taxonomy.contentbean");
+        final Taxonomies taxonomies = taxonomyManager.getTaxonomies();
+
+        addGlobalTaxonomyMapIntoModel(request, locale, taxonomies, publicationLandingPage);
+        addPublicationTypeMapIntoModel(request, locale, taxonomies, publicationLandingPage);
+        addPublicationTopicsMapIntoModel(request, locale, taxonomies, publicationLandingPage);
+    }
+
+    private void addGlobalTaxonomyMapIntoModel(HstRequest request, Locale locale, Taxonomies taxonomies, PublicationLandingPage publicationLandingPage) {
```

Comment:
Could we rename it to `addPublicationProfessionMapIntoModel(...)` to be inline with other methods ? thanks

Reply:
Will do